### DISCUSSION TITLE
Missing smtp.key and smtp.cert

### DISCUSSION THREAD

**Discussion by sinnbeck** [postalserver/postal#2090] [Help with using Postal] [answered]
I have installed postal following the guide and it is working. But now I want to enable tls but the cert files are missing. Did I miss a step somewhere? The postal initialize creates some of the files in the config directory, but not smtp.key and smtp.cert

Anyone know how to create them?

**Comment by sinnbeck** [comment, marked as answer]
So from a lot of googling it seems that I need to provide these myself, and when using caddy the suggestion is to add this to the config, delete the old files, and restart caddy.
```
{
  key_type rsa4096
}
```
I hope this is the correct way :)

**Reply by willpower232** [reply]
Yep, this is the caddy solution. You could provide other certificates but its less confusing for your future self to only have one certificate to manage. Just remember that the lets encrypt certificates expire every few months so there will need to be some restarts of Postal to apply the updated files.
